### ISSUE TITLE
What is the final training result of asynchronous synchronous parallel distributed training？

### ISSUE THREAD

**Issue by gogogwwb** [tensorflow/tensorflow#59740]
<details><summary>Click to expand!</summary> 
 
 ### Issue Type

Support

### Have you reproduced the bug with TF nightly?

Yes

### Source

source

### Tensorflow Version

tf2.8

### Custom Code

Yes

### Current Behaviour?

```shell
When distributed training adopts asynchronous synchronous parallel, the parameters updated by each worker are inconsistent. Is the final result of distributed training the slowest parameter updated by workers?
```

### Standalone code to reproduce the issue

```shell
When distributed training adopts asynchronous synchronous parallel, the parameters updated by each worker are inconsistent. Is the final result of distributed training the slowest parameter updated by workers?
```

### Relevant log output

_No response_</details>

**Comment by tiruk007** [comment]
@gogogwwb 
In sync training, all workers train over different slices of input data in sync, and aggregating gradients at each step. In async training, all workers are independently training over the input data and updating variables asynchronously. Typically sync training is supported via all-reduce and async through parameter server architecture. The final training result of async and sync parallel distributed training depends upon specific implementation, optimization, and training algorithm. Please refer to the [doc_1](https://www.tensorflow.org/guide/distributed_training#types_of_strategies) and [doc_2](https://www.tensorflow.org/tutorials/distribute/multi_worker_with_keras#choose_the_right_strategy) for your reference  and let us know if it helps.

Thank you !

**Comment by tiruk007** [comment]
Closing as stale. Please reopen if you'd like to work on this further.

 Thanks
